packages/web-components/src/components/masthead/masthead-l1.ts

```diff
@@ -42,35 +94,413 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
           `}
-      <dds-top-nav-l1> L1 TOP NAV. </dds-top-nav-l1>
+      <div class="${prefix}--masthead__l1-menu-container-outer">
+        <button
+          class="${prefix}--masthead__l1-menu-container-scroller"
+          id="scroll-prev"
+          @click=${scrollL1TopNav}>
+          ${direction === 'ltr' ? CaretLeft20() : CaretRight20()}
+        </button>
+        <div class="${prefix}--masthead__l1-menu-container-inner">
+          <ul class="${prefix}--masthead__l1-menu">
+            ${(menuItems ?? []).map((menuItem) =>
+              this._renderL1TopNavDropDowns(menuItem)
+            )}
+          </ul>
+        </div>
+        <button
+          class="${prefix}--masthead__l1-menu-container-scroller"
+          id="scroll-next"
+          @click=${scrollL1TopNav}>
+          ${direction === 'ltr' ? CaretRight20() : CaretLeft20()}
+        </button>
+      </div>
+      ${login && login.url && login.title
+        ? html`
+            <a
+              class="${prefix}--masthead__l1-login"
+              href="${ifDefined(login.url)}"
+              >${login.title}</a
+            >
+          `
+        : ''}
+      ${cta && cta.url && cta.title
+        ? html`
+            <a class="${prefix}--masthead__l1-cta" href="${ifDefined(cta.url)}"
+              >${cta.title}</a
+            >
+          `
+        : ''}
+    `;
+  }
+
+  /**
+   * Renders L1MenuItems with subsections containing announcements, headings, links, and footers.
+   *
+   * @param {_L1MenuItem} menuItem Array of menu items containing subnavs
+   * @returns {_TemplateResult} rendered output
+   */
+  protected _renderL1TopNavDropDowns(menuItem) {
+    const {
+      _toggleSubsection: toggleSubsection,
+      _handleTopNavFocusIn: handleTopNavFocusIn,
+    } = this;
+    const { title, url, submenu } = menuItem;
+
+    if (!submenu && url) {
+      return html`
+        <li>
+          <a
+            class="${prefix}--masthead__l1-item"
+            href="${url}"
+            @focusin=${handleTopNavFocusIn}
+            >${title}</a
+          >
+        </li>
+      `;
+    } else if (!submenu) {
+      return html`
+        <li>
+          <span
+            class="${prefix}--masthead__l1-item"
+            @focusin=${handleTopNavFocusIn}
+            >${title}</span
+          >
+        </li>
+      `;
+    }
+
+    const { announcement, menuSections, footer, columns } = submenu ?? {};
+
+    // 3-column dropdown has option for 33/66 split.
+    const hasWideColumn =
+      columns === 3
+        ? menuSections.reduce((hasSpanTwo, section) => {
+            return hasSpanTwo || section.span > 1;
+          }, false)
+        : false;
+
+    // Split can be 33/66 or 66/33.
+    const wideColumnIndex = menuSections[0].span > 1 ? 1 : 2;
+
+    const wideColumns = menuSections.filter((section) => section.span > 1);
+    const normalColumns = menuSections.filter((section) => section.span === 1);
```

Comment:
Sections might get dropped here if they're missing a `span` or have an unexpected value.
```suggestion
    const normalColumns = menuSections.filter((section) => section?.span <= 1);
```